base/resolver_node_lookup.go

```diff
@@ -0,0 +1,127 @@
+// Copyright 2015 The Cockroach Authors.
```

Comment:
Yup. Ben proposed a gossip/resolver/ package which sounds much more sensible.
I'll move it there and rename the files to just plain "node_lookup" and "socket".